app/gunicorn.conf.py

```diff
@@ -0,0 +1,21 @@
+"""
+Configuration file for the Gunicorn server used to run the application in production environments.
+
+Attributes:
+    bind(str): The socket to bind. Formatted as '0.0.0.0:$PORT'.
+    threads(int): The number of threads per worker for handling requests.
+
+For more information, see https://docs.gunicorn.org/en/stable/configure.html
+"""
+
+from src.app_config import AppConfig
+
+app_config = AppConfig()
+
+bind = app_config.host + ':' + str(app_config.port)
```

Comment:
Since we're using the `-b 0.0.0.0:8000` argument when running in the Docker environment, is this `bind` variable only used when not using Docker?
Another related question is: does `host` and `port` need to be part of `app_config` if that configuration is always overridden by the `-b 0.0.0.0:8000` argument?

Reply:
The variable should be used when we're not using Docker. The host and port can change based on the env params.